features/emp-selector/EMPs.ts

```diff
@@ -7,7 +7,8 @@ export const EMPs: { [networkId: number]: string[] } = {
     "0xc0b19570370478EDE5F2e922c5D31FAf1D5f90EA", // uUSDrBTC-OCT
   ],
   42: [
-    "0xFb70A4CBD537B36e647553C279a93E969b041DF0", //"Perpetual", yUSD Kovan Sep30
+    "0x3366b8549047C66E985EcC43026ceD3E831e46A9", // uUSDrBTC Kovan Sep20
```

Comment:
Deployed a new EMP that expires  soon after deployment so I can test out redemptions and cancel withdrawal requests post-expiry